samples/graph-channel-lifecycle/nodejs/Readme.md

```diff
@@ -0,0 +1,56 @@
+# Channel life cycle using Node js
+
+This sample show sthe [Channel](https://docs.microsoft.com/en-us/graph/api/resources/channel?view=graph-rest-1.0) life cycle which includes create, update delete a channel
+
+
+## Prerequisites
+
+- [NodeJS](https://nodejs.org/en/)
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+
+## To try this sample
+
+1) Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+1) In a terminal, navigate to `samples/graph-channel-lifecycle/nodejs
+`
+
+1) Install modules
+
+    ```bash
+    npm install
+    ```
+
+1) Run ngrok - point to port 3978
+
+    ```bash
+    ngrok http -host-header=rewrite 3978
+    ```
+
+
+1) Update the `.env` configuration for the bot to use the ```Microsoft App Id``` and ```App Password``` from the Bot Framework registration. (Note the App Password is referred to as the "client secret" in the azure portal and you can always create a new client secret anytime.)
```

Comment:
This sample does not use Bot, could you please correct this?

Reply:
corrected.